### ISSUE TITLE
Add functionality to modify plots (and their look)

### ISSUE THREAD

**Issue by jankaWIS** [pandas-profiling/pandas-profiling#724]
**Missing functionality**
I have searched pandas profiling documentation, the [advanced usage](https://pandas-profiling.github.io/pandas-profiling/docs/master/rtd/pages/advanced_usage.html) webpage and also the [plotting](https://pandas-profiling.github.io/pandas-profiling/docs/master/rtd/pages/api/_autosummary/pandas_profiling.visualisation.plot.html#module-pandas_profiling.visualisation.plot) but I could not find how to customise plots that it generates. Eg. now it shows correlation matrices with a colorbar having blue as 1 and red as -1. It would be great if one could play with the plots (eg pass some arguments to the underlying matplotlib) to change the colours, the palette, the labels/legend, ...

**Proposed feature**
Be able to add eg `cmap` argument or `color` argument to the plotting function to change the look of the graphs. The same way as one can skip some correlations using the advanced options for usage.

Thanks, your work is really great!

**Comment by sbrugman** [comment]
It's already possible to change the cmap, however it's not directly documented (you can find the related [parameters](https://github.com/pandas-profiling/pandas-profiling/blob/master/src/pandas_profiling/config_default.yaml#L120) in the config file). Feel free to extend the docs with the relevant settings in a PR.

**Comment by jankaWIS** [comment]
Perfect, thanks a lot. How detailed should it be? Do you have some guidelines on how to write your docs? Or is it enough to load some dataset and show on some examples how to make it, eg. as [seaborn](https://seaborn.pydata.org/tutorial/relational.html) does.

**Comment by sbrugman** [comment]
The addtions should be consistent with the existing docs and helpful to readers. You can find the relevant page here: https://github.com/pandas-profiling/pandas-profiling/blob/develop/docsrc/source/pages/advanced_usage.rst. Feel free to add in examples or anything you feel is useful to the reader. Much appreciated!

**Comment by jankaWIS** [comment]
I have updated the docs (see [here](https://github.com/pandas-profiling/pandas-profiling/pull/742)) but I have noticed a few problems when testing it. 

1. For the `histogram`, both `max_bins` and `x_axis_labels` do not exist and raise:
```python
ValueError: Config parameter "max_bins" does not exist.
```
and the last, `bins`, seems to change nothing.

2. It is still not possible to pass an argument to the plot itself, eg. what if I want to have the plots red? Or if I would want to adjust the interaction plots? Anyhow.

**Comment by sbrugman** [comment]
This PR introduces improved config: https://github.com/pandas-profiling/pandas-profiling/pull/751. We can introduce additional plotting parameters easily. Feel free to contribute.

**Comment by jankaWIS** [comment]
I have updated the docs (see [here](https://github.com/pandas-profiling/pandas-profiling/pull/742)) but I have noticed a few problems when testing it. 

1. For the `histogram`, both `max_bins` and `x_axis_labels` do not exist and raise:
```python
ValueError: Config parameter "max_bins" does not exist.
```
and the last, `bins`, seems to change nothing.

2. It is still not possible to pass an argument to the plot itself, eg. what if I want to have the plots red? Or  if I would want to adjust the interaction plots? Anyhow.
